src/main/java/com/dynatrace/prototype/domainModel/KeptnCloudEvent.java

```diff
@@ -40,8 +52,50 @@ public String getSource() {
         return source;
     }
 
-    public String getType() {
-        return type;
+    @JsonProperty(value = "type")
+    public String getFullEventType() {
+        return fullEventType;
+    }
+
+    @JsonIgnore
+    public String getTaskName() {
+        return getSpecificMetaData(KeptnCloudEventValidator.TASK_NAME);
+    }
+
+    @JsonIgnore
+    public String getPlainEventType() {
+        return getSpecificMetaData(KeptnCloudEventValidator.EVENT_TYPE);
+    }
+
+    @JsonIgnore
+    public String getStageName() {
+        return getSpecificMetaData(KeptnCloudEventValidator.STAGE_NAME);
+    }
+
+    @JsonIgnore
+    public String getSequenceName() {
+        return getSpecificMetaData(KeptnCloudEventValidator.SEQUENCE_NAME);
+    }
+
+    /**
+     * Returns the value of the given field from the metaData HashMap if successful or else null.
```

Comment:
```suggestion
     * Returns the value of the given field from the metaData HashMap if successful, otherwise null.
```
Also in the return jdoc